client/src/pages/ModalBookingScreen.js

```diff
@@ -17,7 +17,12 @@ import {
 	Dropdown,
 } from "semantic-ui-react";
 
-function ModalBookingScreen({ bookingDate, refreshBooking, desks }) {
+function ModalBookingScreen({
+	bookingDate,
+	refreshBooking,
+	desks,
+	bookingsByDate,
```

Comment:
I think here it should be `bookings`. Or `bookingsForDate`, not `bookingsByDate` - bookingsByDate is the name of a lookup object where you can put a date in and get an array of bookings

Reply:
I see!